src/main/java/net/hollowcube/luau/LuaStateImpl.java

```diff
@@ -64,15 +64,15 @@ public LuaStateImpl(@NotNull MemorySegment L, @NotNull Arena arena, boolean isTh
         // We implement the user data setter/getter in plain java so this is fine.
         this.ref = GlobalRef.newref(this);
         lua_setthreaddata(L, MemorySegment.ofAddress(this.ref));
+
+        if(!isThread){
```

Comment:
```suggestion
        if (!isThread) {
```

Reply:
I'd not noticed the formatting - I'll run it through IntelliJ's formatter.